src/pdf/ShoppingList/dataPreparation.ts

```diff
@@ -0,0 +1,241 @@
+import {
+    fetchClients,
+    fetchFamilies,
+    fetchLists,
+    fetchParcels,
+} from "@/pdf/ShoppingList/databaseFetch";
+import { Schema } from "@/supabase";
+import { Person } from "@/components/Form/formFunctions";
+
+export interface BlockProps {
+    [key: string]: string;
+}
+
+interface ParcelInfoAndClientID {
+    parcelInfo: ParcelInfo;
+    clientID: string;
+}
+
+interface ClientDataAndFamilyData {
+    clientData: Schema["clients"];
```

Comment:
How come client data isn't an array but family data is?

Reply:
each parcel is only linked to 1 client, but each client can have multiple family members (one member takes up one row)